legend-engine-xts-json/legend-engine-xt-json-pure/src/main/resources/core_external_format_json/executionPlan/tests/simple.pure

```diff
@@ -40,6 +40,20 @@ function <<meta::pure::profiles::test.Test, meta::pure::profiles::test.AlloyOnly
   assertEquals('{"firstName":"John","lastName":"Doe"}', $result);
 }
 
+function <<meta::pure::profiles::test.Test, meta::pure::profiles::test.AlloyOnly>> meta::external::format::json::executionPlan::test::simple::testSimpleJsonQueryWithFunctionalInput(): Boolean[1]

```

Comment:
Can we have one test with let function doing the decodeUrl part

Reply:
Based on discussion, this scenario is already covered